java11/src/main/java/feign/http2client/Http2Client.java

```diff
@@ -124,6 +128,9 @@ protected Response toFeignResponse(Request request, HttpResponse<byte[]> httpRes
     final OptionalLong length = httpResponse.headers().firstValueAsLong("Content-Length");
 
     return Response.builder()
+        .protocolVersion(httpResponse.version().name()
```

Comment:
this seems to fragile.

BTW, can you add a comment showing that is the expect value for `version()`

Also, I don't think it would be too back to have protocol = `HTTP_1_1`.

If you reallt think this name transformation is important, can we change this to a `switch` and hardcoded responses, just to make it easier to read.

Reply:
I agree.
I have replaced the string regexp-building by looking for proper enum value by its `name()` and `toString()` values. This solution covers all available in **java11** values: `HTTP_1_1` and `HTTP_2`.